metadata.json

```diff
@@ -1,6 +1,6 @@
 {
 "shell-version": [
-    "45"
+    "46"
```

Comment:
We should still try to support 45 if the efforts are not big enough.

Reply:
Fixed.